test_serve.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from   serve import app
+
+NEXIST_DANDI_ID = "999999"
+
+@pytest.mark.parametrize('req_url,resp_url', [
+    ("/", "https://gui.dandiarchive.org/"),
+    ("/about", "https://www.dandiarchive.org"),
+    ("/dandiset", "https://gui.dandiarchive.org/#/dandiset"),
+    (
+        "/dandiset/000003",
+        "https://gui.dandiarchive.org/#/dandiset/5e6eb2b776569eb93f451f8d",
```

Comment:
@mgrauer - this is going to change once publish is merged right?

Reply:
Yes, it should be like `https://gui.dandiarchive.org/#/dandiset/000003`.

This change will come from #15, which we will merge when the publish branch is merged in dandiarchive.